### PR TITLE
Add Gazami

### PR DESCRIPTION
**Pull request by SpiddlTV**
## Type of change

- [x] Add new directory for a server

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

## Further comments
Hope I did everything right.

### CODE REVIEW

**Comment by toohard2explain** [comment]
every single file is missing?

**Comment by SpiddlTV** [comment]
> every single file is missing?

Yeah, I couldnt create another commit so I asked on the Discord bot got no answer yet

**Comment by maaaathis** [comment]
compress your images

**Comment by SpiddlTV** [comment]
They should be

**Comment by SpiddlTV** [comment]
Oh no wait I changed them one moment

File: minecraft_servers/gazami/manifest.json
Comment (maaaathis) [inline minecraft_servers/gazami/manifest.json:1]: fix your indentations

File: minecraft_servers/gazami/manifest.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+   "brand":{
+      "primary":"#004d54",
+      "background":"#1a1919",
+      "text":"#FFFFFF"
+   },
+    "discord": {
+        "server_id": 735191006235852860,
+        "rename_to_minecraft_name": false
+    },
+    "docs_page": "https://docs.gazami.de"
```
Comment (maaaathis) [inline minecraft_servers/gazami/manifest.json:49]: `docs_page` is not a valid prop

File: minecraft_servers/gazami/manifest.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    "server_wildcards": [
+        "gazami.de",
+        "gazami.eu"
+    ],
+    "supported_languages": [
+        "de"
+    ],
+    "social": {
+        "web": "https://gazami.de",
+        "threads": "gazami.de",
```
Comment (maaaathis) [inline minecraft_servers/gazami/manifest.json:14]: `social.threads` is also not a valid prop. See https://github.com/LabyMod/server-media/blob/master/docs/Manifest.md#available-keys

Comment (SpiddlTV) [reply minecraft_servers/gazami/manifest.json:14]: Changed

Comment (SpiddlTV) [reply minecraft_servers/gazami/manifest.json]: Now its changed

File: minecraft_servers/gazami/manifest.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
+{
+    "server_name": "gazami",
+    "nice_name": "Gazami",
+    "direct_ip": "gazami.de",
+    "server_wildcards": [
+        "gazami.de",
```
Comment (DoJapHD) [inline minecraft_servers/gazami/manifest.json:6]: Not a wildcard

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Comment by DoJapHD** [comment]
As [maaaathis](https://github.com/maaaathis) already mentioned, please fix your indentations. You can do that f.e. by using a JSON formatter

**Comment by SpiddlTV** [comment]
> As [maaaathis](https://github.com/maaaathis) already mentioned, please fix your indentations. You can do that f.e. by using a JSON formatter

Now it should be absolutely right

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**